ci/integration-tests/README.md

```diff
@@ -0,0 +1,43 @@
+# Integration Tests
+
+e2e integration tests with gas profiling.
+
+`cd ci/integration_tests && cargo t` to run all tests.
+
+`cargo t fn_test_name` to run individual integration tests.
```

Comment:
Also some `CONFIG` variable needs to be set.
https://github.com/CosmWasm/cw-nfts/pull/84/files#diff-1d721ea28fc40ef113f3080306e4595d8176b794321429300530a386a0e305c6R83

and I don't see any mention about that (I might be blind though).